src/components/Breakpoints/Breakpoints.stories.mdx

```diff
@@ -0,0 +1,48 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import cssVariables from "../../helpers/CSSVariablesHelper";
+import getCSSVariable from "../../helpers/getCSSVariable";
+import Heading from "../Heading/Heading";
+import { CSSVariablesInterface } from "../../interfaces";
+import { getCategory } from "../../utils/componentCategories";
+import * as stories from "./Breakpoints.stories.tsx";
+
+<Meta
+  title={getCategory("Breakpoints")}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=10975%3A16",
```

Comment:
With this type of documentation page, the figma addon doesn't show up.

Reply:
OK.  Gone.